### PR TITLE
make the Benefits page readable: add Converters TOC

### PR DESCRIPTION
**Pull request by christinerogers**
To upgrade the Benefits page from an unfriendly scroll journey --

This PR adds internal links in the Converters section and a mini-TOC for them

### CODE REVIEW

File: _pages/benefits.md [hunk 1]
```diff
@@ -12,14 +12,31 @@ By using this standard you will benefit in the following ways:
 
 ## Converters
 
+Tools which convert data to/from BIDS : 
+* [Data converters](#data-converters)
+* [MEEG converters](#meeg-converters)
+* [from BIDS converters](#from-bids-converters)
+* [Physio converters](#physio-converters)
+* [Other converters](#other-converters)
```
Comment (sappelhoff) [inline _pages/benefits.md:20]:
```suggestion
Software tools to convert data to or from BIDS:

- [MRI and PET converters](#mri-and-pet-converters)
- [EEG, MEG, iEEG converters](#eeg-meg-ieeg-converters)
- ['From BIDS' converters](#from-bids-converters)
- [Physiological data converters](#physiological-data-converters)
- [Miscellaneous](#miscellaneous)
```

File: _pages/benefits.md [hunk 2]
```diff
@@ -12,14 +12,31 @@ By using this standard you will benefit in the following ways:
...
 
+### Physio converters
+
 {% include converters_table.html members=site.data.physio_converters %}
 
+### Other converters
+
 {% include converters_table.html members=site.data.other_converters %}
 
 # Software currently supporting BIDS:
```
Comment (sappelhoff) [inline _pages/benefits.md:42]:
```suggestion
{% include converters_table.html members=site.data.converters %}

{% include converters_table.html members=site.data.MEEG_converters %}

{% include converters_table.html members=site.data.from_BIDS_converters %}

{% include converters_table.html members=site.data.physio_converters %}

{% include converters_table.html members=site.data.other_converters %}

# Software currently supporting BIDS:
```

**Review by sappelhoff — COMMENTED** [review]
+1 for the TOC, but let's use the headings that are automatically generated -- with the current changes (without my suggestions), we'd have duplicates, see: https://deploy-preview-369--astounding-beignet-8ca2d6.netlify.app/benefits#data-converters

image

**Comment by Remi-Gau** [comment]
probably worth revisiting this PR to see if what's main looks "good enough"

https://bids-website.readthedocs.io/en/latest/tools/converters/

**Comment by christinerogers** [comment]
I think we've lost something with the subpages here.  -- the ability to search the page for (e.g.) "EEG" since a lot of tools are under one category but could be under multiple. 
is it easy to add a "search all converters" box/feature or is a table of contents left-side still with a long scroll in the main panel worth trying?

**Comment by Remi-Gau** [comment]
Closing as this is now obsolete on the new website

**Pull request closed without merging**